kafka-java-maven/src/main/java/org/example/KafkaConfig.java

```diff
@@ -0,0 +1,24 @@
+package org.example;
+
+import org.apache.kafka.clients.producer.ProducerConfig;
+
+import java.util.Properties;
+
+public class KafkaConfig {
+
+    static Properties properties() {
+        var properties= new Properties();
+
+        properties.setProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, System.getenv("KAFKA_HOST"));
```

Comment:
[nitpick] it would be nice to group together the read of env vars as we did in Python, it reads really well :-)